src/Helper/HtmlForm.php

```diff
@@ -51,7 +51,7 @@ public static function getAttributeValue(FormModelInterface $form, string $attri
      */
     public static function getInputId(FormModelInterface $form, string $attribute, string $charset = 'UTF-8'): string
     {
-        $name = mb_strtolower(static::getInputName($form, $attribute), $charset);
+        $name = (string) mb_strtolower(static::getInputName($form, $attribute), $charset);
```

Comment:
I don't think it can return non-string.

Reply:
> I don't think it can return non-string.

mb_strtolower return false and generate a warning if is called with undefined charset (i.e. mb_strtolower('example', 'unkownCharset') )

Reply:
Why should we cast this `false` to a string?